Add quilt3 login FAQ (#3099)
quiltdata/quilt: docs/FAQ.md

You can then follow the above with `q3.Package.delete(pname, registry=reg, top_hash=p.top_hash)`.

## Do I have to login via quilt3 to use the Quilt APIs? How do I push to Quilt from a headless environment like a Docker container?

> If you have configured your [AWS command line](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) 
> you can run `aws configure` and use your valid AWS credentials to push from your
> machine or from any pipeline. Remember to run `quilt3 logout` if you were logged 
> in to your existing Quilt stack!

Reviewer: ```suggestion
Configure [AWS CLI credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) 
and `quilt3` will use the same for its API calls. Select among multiple profiles in your shell as follows:
```bash
export AWS_PROFILE=your_profile
```

> Be sure to `quilt3 logout` if you were previously logged in to your existing Quilt stack and wish to use
AWS credentials instead.

> The necessary AWS S3 object permissions are similar to 
> [this bucket policy](https://docs.quiltdata.com/advanced/crossaccount#bucket-policies) 
> but you do not need either the `s3:GetBucketNotification` or `s3:PutBucketNotification` 
> to use `quilt3`.

Reviewer: ```suggestion
The S3 permissions needed by `quilt3` are similar to
[this bucket policy](https://docs.quiltdata.com/advanced/crossaccount#bucket-policies)
but `quilt3` does not need either `s3:GetBucketNotification` or `s3:PutBucketNotification`.
```

You can then follow the above with `q3.delete_package(pname, registry=reg, top_hash=p.top_hash)`.

## Do I have to login via quilt3 to use the Quilt APIs? How do I push to Quilt from a headless environment like a Docker container?

Configure [AWS CLI credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) 
and `quilt3` will use the same for its API calls. Select among multiple profiles in your shell as follows:
```bash
export AWS_PROFILE=your_profile

> ```

Reviewer: Add a note on "Be sure to run `quilt3 logout` if you've previously logged in."

Author: Added

You can then follow the above with `q3.delete_package(pname, registry=reg, top_hash=p.top_hash)`.

## Do I have to login via quilt3 to use the Quilt APIs? How do I push to Quilt from a headless environment like a Docker container?

Configure [AWS CLI credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) 
and `quilt3` will use the same for its API calls.

> Note: be sure to run `quilt3 logout` if you've previously logged in.

Reviewer: ```suggestion
> Be sure to run `quilt3 logout` if you've previously logged in.
```
